lib/AcqUnitFilter/AcqUnitFilterContainer.js

```diff
@@ -0,0 +1,29 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+
+import { getAcqUnitsOptions } from '../utils';
+import { acqUnitsManifest } from '../manifests';
+import AcqUnitFilter from './AcqUnitFilter';
+
+const AcqUnitFilterContainer = ({ resources, ...rest }) => {
+  const acqUnitsOptions = useMemo(() => getAcqUnitsOptions(resources.acqUnits), [resources.acqUnits]);
```

Comment:
AFAIK, there is `resources.acqUnits?.records`

Reply:
👌